api/src/main/java/org/openmrs/api/impl/PatientServiceImpl.java

```diff
@@ -1636,4 +1726,25 @@ public void checkIfPatientIdentifierTypesAreLocked() {
 			throw new PatientIdentifierTypeLockedException();
 		}
 	}
+	
+	public void addPatientMergeAction(PatientMergeAction patientMergeAction) {
+		if (this.patientMergeActions == null) {
+			this.patientMergeActions = new ArrayList<PatientMergeAction>();
+		}
+		this.patientMergeActions.add(patientMergeAction);
+	}
+	
+	@Override
+	public void removePatientMergeAction(PatientMergeAction patientMergeAction) {
+		if (this.patientMergeActions == null) {
+			this.patientMergeActions = new ArrayList<PatientMergeAction>();
```

Comment:
and this into a constructor? Or maybe the arraylist can be created at declaration.